src/test/datascience/group1.nonConda.functional.test.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+'use strict';
+import * as assert from 'assert';
+import { Disposable } from 'vscode';
+
+import { InteractiveWindowMessages } from '../../client/datascience/interactive-common/interactiveWindowTypes';
+import { DataScienceIocContainer } from './dataScienceIocContainer';
+import { takeSnapshot, writeDiffSnapshot } from './helpers';
+import { addCode, getOrCreateInteractiveWindow, runTest } from './interactiveWindowTestHelpers';
+import { CellPosition, verifyHtmlOnCell } from './testHelpers';
+// tslint:disable-next-line: no-require-imports no-var-requires
+
+// tslint:disable:max-func-body-length trailing-comma no-any no-multiline-string
+suite('DataScience Interactive Window output tests', () => {
+    const disposables: Disposable[] = [];
+    let ioc: DataScienceIocContainer;
+    let snapshot: any;
+
+    suiteSetup(function () {
+        snapshot = takeSnapshot();
+        if (process.env.CI_PYTHON_PATH?.includes('conda')) {
+            console.log('This test should only run without conda installed');
+            this.skip();
+        }
+    });
+    setup(async () => {
+        ioc = new DataScienceIocContainer();
+        ioc.registerDataScienceTypes();
+        return ioc.activate();
+    });
+
+    suiteTeardown(() => {
+        writeDiffSnapshot(snapshot, 'Interactive Window');
+    });
+
+    teardown(async () => {
+        for (const disposable of disposables) {
+            if (!disposable) {
+                continue;
+            }
+            // tslint:disable-next-line:no-any
+            const promise = disposable.dispose() as Promise<any>;
+            if (promise) {
+                await promise;
+            }
+        }
+        await ioc.dispose();
+    });
+
+    function verifyHtmlOnInteractiveCell(html: string | undefined | RegExp, cellIndex: number | CellPosition) {
+        const iw = ioc.getInteractiveWebPanel(undefined).wrapper;
+        iw.update();
+        verifyHtmlOnCell(iw, 'InteractiveCell', html, cellIndex);
+    }
+
+    runTest(
+        'Simple text with no python extension',
```

Comment:
Seems jupyter extension will not run a 'conda' kernel if there's no python extension there to get it to work. So these tests will only run if there's a plain python. I moved them to a separate file to indicate this.

File name also forces the group number for the functional test split.